app/gimlet-rot/app-b.toml

```diff
@@ -125,9 +139,13 @@ spi_num = 5
 [tasks.sp_measure]
 name = "task-sp-measure"
 priority = 5
-max-sizes = {flash = 32768, ram = 8192}
+max-sizes = {flash = 131072, ram = 8192}
 task-slots = ["swd"]
 stacksize = 2048
```

Comment:
Given sp_measure was always supposed to be a sample task I'm not sure if it's worth it to increase the flash size vs just removing it if actually requires more flash.

Reply:
I made this consistent with other app.toml files. I should have left it alone, but it can be a clean-up PR later.